core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/Parameter.java

```diff
@@ -40,8 +40,8 @@
 	private String required;
 	@JsonProperty
 	private String defaultValue;
-	@JsonProperty
-	private String choice;
+	@JsonProperty("enum")
+	private String[] choice;
```

Comment:
please note that the attribute cannot be named "enum" so keep "choice" which was the name used previously